controller/search.js

```diff
@@ -28,6 +30,7 @@ function setup( apiConfig, esclient, query, should_execute ){
 
     // if there's no query to call ES with, skip the service
     if (_.isUndefined(renderedQuery)) {
+      debugLog.stopTimer(req, 'No query to call ES with. Skipping');
```

Comment:
@tigerlily-he what's happens if you have two timers running concurrently? If the message is optional then running two service requests concurrently could look like this:

```bash
01s Timer Began
01s Timer Began
02s Timer Stopped
99s Timer Stopped
```

and you wouldn't know which was which?

It could be as simple as just discouraging people from leaving the message blank.

another option is to store each timer as a variable, such as:

```javascript
var timer = debugLog.beginTimer(req, 'Msg 1');

... timely code

timer.end();
```